ng-dev/pr/merge/config.ts

```diff
@@ -85,49 +84,29 @@ export interface MergeConfig {
  * the dev-infra configuration is loaded as that could slow-down other commands.
```

Comment:
@josephperrott can we update that comment? it might be worth preserving the contents in some way, but saying that we do not support custom configuration, not sure. can be a follow-up too.

Reply:
I just removed it entirely as we no longer even need that type.